dask/tests/test_base.py

```diff
@@ -284,3 +284,19 @@ def test_visualize():
         dsk = {'a': 1, 'b': (add, 'a', 2), 'c': (mul, 'a', 1)}
         visualize(x, dsk, filename=os.path.join(d, 'mydask.png'))
         assert os.path.exists(os.path.join(d, 'mydask.png'))
+
+
+def test_use_cloudpickle_to_tokenize_functions_in__main__():
+    import sys
+
+    defn = """
```

Comment:
use `textwrap.dedent` to make this more readable: https://docs.python.org/2/library/textwrap.html#textwrap.dedent